includes/site-health/load.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Site Health tests loader.
```

Comment:
Nit-pick: Those are commonly referred to as "checks", not "tests". The latter could be confused with unit test coverage.
```suggestion
 * Site Health checks loader.
```